### ISSUE TITLE
Unable to chain `find_by_text`

### ISSUE THREAD

**Issue by garrettedwards95** [cobrateam/splinter#426]
I pip install'd from master so that I could get the newest `find_by_text` changes, which is great, however I find that I am unable to chain any `find_by_*` into `find_by_text`:

```python
import splinter
browser.visit('http://google.com/')
browser.find_by_css('div[id="viewport"]').find_by_text('Gmail')
```

Here we encounter the error:
```
Traceback (most recent call last):
  File "/Users/garrettedwards/auto_integration/tools/pythontest/testing.py", line 43, in <module>
    browser.find_by_css('div[id="viewport"]').find_by_text('Gmail')
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/splinter/driver/webdriver/__init__.py", line 597, in find_by_text
    original_query=text)
TypeError: find_by_xpath() got an unexpected keyword argument 'original_find'
```

It just looks like there are some leftover arguments in the definition of `find_by_text`

```python
def find_by_text(self, text):
    selector = '//*[text()="%s"]' % text
    return self.find_by_xpath(selector, original_find='text',
                              original_query=text)
```

**Comment by garrettedwards95** [comment]
Great! Started using this already.
Thanks.

**Comment by andrewsmedina** [comment]
@garrettedwards95 released in 0.7.3 :)
